.github/workflows/test.yml

```diff
@@ -26,3 +26,14 @@ jobs:
             - run: npm run lint
             - run: npm audit --audit-level=critical
             - run: npm run build-with-tests && npm run test-transpiled
+            - name: Extract k8s version
+              id: extract_version
+              run: |
+                  version=$(grep 'export KUBERNETES_BRANCH=' settings | cut -d '"' -f 2)
+                  echo "setting kubernetes version to $KUBERNETES_V_STRING"
+                  echo "KUBERNETES_V_STRING=$version" >> "$GITHUB_ENV"
+            - name: start minikube
```

Comment:
This appears to be failing the CI. Do we need these changes?

Reply:
they aren't critical, but i'd like to add some minikube integration tests. i can move that to a separate PR